_usage/command-line.md

```diff
@@ -190,10 +194,12 @@ works with MS SQL as well as PostgreSQL.
 
 ### Using psql to connect
 
-If you don't want to use tsql as your Linux command line interface, you can use a
-modified version of psql which comes as part of the Babelfish package. Note:
-This HAS to be the Babelfish version of psql - otherwise the following feature
-will NOT work.
+You can also use PostgreSQL's `psql` to connect to Babelfish. However, this is
+**strongly** discouraged, and the behavior will be unpredictable. It is
+therefore not recommended to to that &ndash; in fact it can be pretty dangerous.
+
+Babelfish comes with a modified version of `psql`, which allows to set
+the SQL dialect:
 
 ```bash
 SET babelfishpg_tsql.sql_dialect = 'tsql';
```

Comment:
REMOVE THIS !!!! 
This is the part that is strongly discourage and can yield unpredictable results. This option exists for development purposes only, and only developers with a deep understanding of the implications can use it, and only for very specific purposes. In other words: don't use it.  Best removethis completely, this will only cause trouble.